src/Components/Aspire.Confluent.Kafka/MetricsChannel.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.Channels;
+
+namespace Aspire.Confluent.Kafka;
+
+internal sealed class MetricsChannel
+{
+    private readonly Channel<string> _channel = Channel.CreateUnbounded<string>(new UnboundedChannelOptions()
```

Comment:
Feels like this should be bounded or we risk creating a memory leak.

Reply:
bounded to arbitrary 10k.